### PR TITLE
Add document for NIC partitioning

### PR DESCRIPTION
**Pull request by pliurh**

### CODE REVIEW

**Comment by pliurh** [comment]
@jboxman @lamek @zshi-redhat  @zhaozhanqi PTAL

File: modules/nw-sriov-configuring-device.adoc [hunk 1]
```diff
@@ -89,3 +89,69 @@ A device can be used in either mode.
...
+
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-net-1
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: net-1
```
Comment (zhaozhanqi) [inline modules/nw-sriov-configuring-device.adoc]: the value for resourceName should not contain '-' character

File: modules/nw-sriov-configuring-device.adoc [hunk 2]
```diff
@@ -89,3 +89,69 @@ A device can be used in either mode.
...
+
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-net-1-dpdk
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: net-1-dpdk
```
Comment (zhaozhanqi) [inline modules/nw-sriov-configuring-device.adoc]: ditto

File: modules/nw-sriov-configuring-device.adoc (hunk 1 above)
Comment (pliurh) [reply modules/nw-sriov-configuring-device.adoc]: Fixed.

File: modules/nw-sriov-configuring-device.adoc (hunk 2 above)
Comment (pliurh) [reply modules/nw-sriov-configuring-device.adoc]: Fixed

File: modules/nw-sriov-configuring-device.adoc [hunk 3]
```diff
@@ -89,3 +89,69 @@ A device can be used in either mode.
...
+. `<LastVF>`   - is the last VF index (0-based) that is included into the range
+
+Example:
+
+The selector for interface named netpf0 and VF 2 upto 7 (included 2 and 7) will look like:
+
+[source,yaml]
+pfNames: ["netpf0#2-7"]
+
+It is allowed to select the VFs from the same PF in different policy CRs, however you have to make sure:
```
[the remark this answers is not in the document]
Comment (pliurh) [reply modules/nw-sriov-configuring-device.adoc]: Yes.

File: modules/nw-sriov-configuring-device.adoc [hunk 4]
```diff
@@ -89,3 +89,69 @@ A device can be used in either mode.
...
+
+. The `numVfs` shall be identical among those policies, if they all select the same PF.
+. The VF index shall be in in range of `0` to `<numVfs>-1`. For example, if you have a policy with `numVfs` 8, then the `<FirstVF>` shall not be smaller than 0, while the `<FirstVF>` shall not larger than 7.
```
[the remark this answers is not in the document]
Comment (pliurh) [reply modules/nw-sriov-configuring-device.adoc]: fixed

File: modules/nw-sriov-configuring-device.adoc [hunk 5]
```diff
@@ -89,3 +89,69 @@ A device can be used in either mode.
...
+. The VFs ranges in different policies shall not be overlapped.
+
```
[the remark this answers is not in the document]
Comment (pliurh) [reply modules/nw-sriov-configuring-device.adoc]: fixed

**Comment by lamek** [comment]
Thanks @pliurh. Is this just for 4.4, or 4.3 as well?

**Comment by lamek** [comment]
@pliurh I bumped this into a new PR b/c I wanted to break the content out into a new file and I did some significant changes. I pinged you in the new PR as well. PTAL.
https://github.com/openshift/openshift-docs/pull/20548

**Comment by pliurh** [comment]
@lamek This is only for 4.4.

**Comment by lamek** [comment]
The new PR I linked above has been merged and cherrypicked, so I'm closing this one. Thanks again.

**Pull request closed without merging**